src/main/kotlin/com/seokjoo/todo/domain/entity/todouser/Money.kt

```diff
@@ -20,4 +20,11 @@ class Money(
         check(this.balance - balance >= 0) { throw TodoException.of(TodoExceptionType.BALANCE_NOT_ENOUGH) }
         return Money(balance = this.balance - balance)
     }
+
+    fun hasNoBalance() = balance == 0L
```

Comment:
```suggestion
    fun needToCharge() = balance == 0L
```

요런 네이밍은 어떠신지,, 가볍게 제안드립니다 ㅎㅡㅎ